vNext/test/NonSiloTests/project.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "dependencies": {
+    "System.Management.Automation": "6.1.7601.17515",
```

Comment:
why is this needed? It might be a legitimate usage for now, but I'd like to understand it before letting it in, as it might mean it won't work in the WALK phase when we run tests in .NET Core.